Project.toml

```
@@ -1,6 +1,6 @@
 name = "DocumenterTools"
 uuid = "35a29f4d-8980-5a13-9543-d66fff28ecb8"
-version = "0.1.17"
+version = "0.1.18"
```

I guess this is going to be
```suggestion
version = "0.1.20"
```
now.  Same in the ChangeLog above